generated/nimodinst/tests/test_modinst.py

```diff
@@ -50,3 +50,25 @@ def test_context_manager(self):
             self.patched_ctypes_library.niModInst_OpenInstalledDevicesSession.assert_called_once_with(b'', ANY, ANY)
             self.patched_errors._handle_error.assert_called_once_with(session, self.patched_ctypes_library.niModInst_OpenInstalledDevicesSession.return_value)
         self.patched_ctypes_library.niModInst_CloseInstalledDevicesSession.assert_called_once_with(SESSION_NUM_FOR_TEST)
+
+    def test_freeze(self):
```

Comment:
test_cannot_add_properties_to_session

Better be more explicit in our test names.